tests/cases/php/closing-tag-end

```diff
@@ -0,0 +1,19 @@
+---DESCRIPTION---
+No closing PHP tags at the end of files
+---CONTENTS---
+<?php
+
+declare(strict_types=1);
+
+$foo = 'bar';
+
+?>
```

Comment:
Will it simply error if there is text/html after a closing tag?

Reply:
Yep. (`closing-tag` covers that. These aren't meant to be exhaustive unit tests.)